src/router/index.ts

```diff
@@ -6,7 +6,7 @@ const routes: Array<RouteRecordRaw> = [
     path: "/",
     name: "Home",
     component: Home,
-  }
+  },
```

Comment:
why is there a comma for nothing after? I will never understand this logic

Reply:
This is just Prettier's default:

* https://github.com/prettier/prettier/issues/68
* https://github.com/prettier/prettier/pull/6963

The rationale is that it reduces unnecessary diff noise when adding trailing entries, I think.

Reply:
I personally think they are unnecessary because my mind is tricky by thinking there is something after since I treat commas as separators. it's like having a comma after the last element of an array

Reply:
Ah, I'm pretty neutral about it: I just care that code formatting is automatic, and I don't have to think about it. 😊️

Reply:
Sorry about it, I will apply the rule to it if you don't mind?

Reply:
We could do that, though I would lean towards sticking to the default prettier config as far as possible, unless we have a strong reason to deviate from it?

Reply:
I think Prettier is "pretty" opinionated, just like that rule was suggested by someone's preference. 
Many [people](https://github.com/prettier/prettier/issues/68#issuecomment-271725357) opposed it.

And I agree with [this comment](https://github.com/prettier/prettier/issues/68#issuecomment-271891128)